.github/workflows/update-web-docs.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2019-2024 vdaas.org vald team <vald@vdaas.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: "Update web contents"
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
Do you want to dispatch every time its merged into main?

Reply:
it will run when pushed commit including changes of target files to the main branch.
Target files are defined as below :)
https://github.com/vdaas/vald/pull/2294/files#diff-cd359f72cefa48e6378208fc449ae174b7246899db892b2888bc8519705df465R28-R30

Reply:
So the web contents will be updated every time those docs get changed?
I think currently the web updates only when there is release so this seems different from the current logic.

I'm fine if this is what you intend to do.

Reply:
> So the web contents will be updated every time those docs get changed?
Yes.

> I think currently the web updates only when there is release so this seems different from the current logic.
Actually, the current update logic has two patterns:
- update contents when a document has been updated because some updates affect the latest version too.
- update contents and static files (e.g., header.html) when released.

But, recently, I haven't been able to update properly :sweat: